synfig-core/src/modules/lyr_freetype/CMakeLists.txt

```diff
@@ -3,9 +3,17 @@ add_library(lyr_freetype MODULE
         "${CMAKE_CURRENT_LIST_DIR}/lyr_freetype.cpp"
 )
 
-target_include_directories(lyr_freetype PRIVATE ${PANGO_INCLUDE_DIRS})
+if (HARFBUZZ_FOUND)
+	target_compile_definitions(lyr_freetype PUBLIC -DHAVE_HARFBUZZ=1)
+endif()
 
-target_link_libraries(lyr_freetype synfig ${CAIRO_LIBRARIES} ${PANGO_LIBRARIES} ${PANGOCAIRO_LIBRARIES} ${FT_LIBRARIES} ${FONT_CONFIG_LIBRARIES})
+if (FRIBIDI_FOUND)
+	target_compile_definitions(lyr_freetype PUBLIC -DHAVE_FRIBIDI=1)
+endif()
+
+target_include_directories(lyr_freetype PRIVATE ${PANGO_INCLUDE_DIRS} ${HARFBUZZ_INCLUDE_DIRS} ${FRIBIDI_INCLUDE_DIRS})
+
+target_link_libraries(lyr_freetype synfig ${CAIRO_LIBRARIES} ${PANGO_LIBRARIES} ${PANGOCAIRO_LIBRARIES} ${FT_LIBRARIES} ${FONT_CONFIG_LIBRARIES} ${FRIBIDI_LIBRARIES})
```

Comment:
Sorry I missed it before.
There is not any dependency on Pango, and no more on Cairo.
Any reason for no reference on HarfBuzz for linking?

Reply:
> Any reason for no reference on HarfBuzz for linking?

No, just missed it because all builds without problem.